pandas/core/generic.py

```diff
@@ -10892,7 +10892,7 @@ def ewm(
         span: float | None = None,
         halflife: float | TimedeltaConvertibleTypes | None = None,
         alpha: float | None = None,
-        min_periods: int = 0,
+        min_periods: int | None = 0,
```

Comment:
is this (and the changed annotation below) orthogonal to the rest of the PR?

Reply:
A bit.

`EWM.min_periods` was typed as `int`

It's parent class typed `min_periods` as `int | None`.

Since I was calling `super` needed to get the types and some code aligned.